README.md

```diff
@@ -0,0 +1,189 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+> This is a mobile application that allows users to add expenditures for easy tracking of their monetary usage. Users need to sign up to use this application.
+
+## 🛠 Built With <a name="built-with"></a>
+ - Ruby on Rails
+ - Postgresql
+
+### Tech Stack <a name="tech-stack"></a>
+- Bootstrap
+- Node
+- Ruby gems
+- Capybara
+- RSpec
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://heroku.com/">Heroku</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+- User authentication
+- Addition of new categories and expenses
+- Side menu
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://stormy-plateau-90686.herokuapp.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+- Clone this repository by writing `git clone https://github.com/Aquinasmoriasi/budget-app.git`
+- Run `bundle install` to install the project's dependencies
+- Add your database username and password and write `rails db:create` to create the database
+- Run  `rails db:migrate` to migrate the tables to the created database
+- Run `rails server` to open the local host
+- To run the tests write `rspec <path to the spec file>` to run the application's test
+
+### Prerequisites
+
+In order to run this project you need:
+- Install PostgreSQL in your machine
+- install Ruby and Ruby on rails
+- Install your preferred IDE(Visual Studio is highly recommended)
+- Then run: 
+
+```
+ bundle install
+
+```
+
+### Setup
+
+- Clone this repository to your desired folder
+- Navigate to the cloned repository and follow the aforementioned steps to get started
+
+### Install
+
+Install this project with:
+
+- PostgreSQL
+- Ruby
+- Ruby on Rails
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+rspec <relative path to the test file>
+```
+
+### Deployment
+
+This project is deployed at [https://stormy-plateau-90686.herokuapp.com/](https://stormy-plateau-90686.herokuapp.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Kindly ensure to add the _Future Features_ section to your ReadMe, which should include 1-3 potential features to be added